traffic_ops/install/bin/input.json

```diff
@@ -1,179 +1,151 @@
-  "/opt/traffic_ops/app/conf/production/database.conf":[
+  "/opt/traffic_ops/app/conf/cdn.conf": [
     {
-      "Database type":"Pg",
-      "config_var":"type"
+      "Generate a new secret?": "yes",
+      "config_var": "genSecret"
     },
     {
-      "Database name":"traffic_ops_db",
-      "config_var":"dbname"
+      "Number of secrets to keep?": "10",
+      "config_var": "keepSecrets"
     },
     {
-      "Database server hostname IP or FQDN":"localhost",
-      "config_var":"hostname"
+      "Number of workers?": "12",
+      "config_var": "workers"
     },
     {
-      "Database port number":"5432",
-      "config_var":"port"
-    },
-    {
-      "Traffic Ops database user":"traffic_ops",
-      "config_var":"user"
-    },
-    {
-      "Traffic Ops database password":"default",
-      "config_var":"password",
-      "hidden":"1"
+      "Traffic Ops url?": "https://[::]",
+      "config_var": "base_url"
     }
   ],
-  "/opt/traffic_ops/app/db/dbconf.yml":[
+  "/opt/traffic_ops/app/conf/ldap.conf": [
     {
-      "Database server root (admin) user":"root",
-      "config_var":"dbAdminUser"
+      "Do you want to set up LDAP?": "no",
+      "config_var": "setupLdap"
     },
     {
-      "Database server admin password":"default",
-      "config_var":"dbAdminPw",
-      "hidden":"1"
+      "LDAP server hostname": "",
+      "config_var": "hostname"
     },
     {
-      "Download Maxmind Database?":"yes",
-      "config_var":"maxmind"
-    }
-  ],
-  "/opt/traffic_ops/app/conf/cdn.conf":[
-    {
-      "Generate a new secret?":"yes",
-      "config_var":"genSecret"
+      "LDAP Admin DN": "",
+      "config_var": "admin_dn"
     },
     {
-      "Number of secrets to keep?":"10",
-      "config_var":"keepSecrets"
+      "LDAP Admin Password": "",
+      "config_var": "password",
+      "hidden": "1"
     },
     {
-      "Port to serve on?":"443",
-      "config_var":"port"
+      "LDAP Search Base": "",
+      "config_var": "search_base"
     }
   ],
-  "/opt/traffic_ops/app/conf/ldap.conf":[
+  "/opt/traffic_ops/app/conf/production/database.conf": [
     {
-      "Do you want to set up LDAP?":"no",
-      "config_var":"setupLdap"
+      "Database type": "Pg",
+      "config_var": "type"
     },
     {
-      "LDAP server hostname":"",
-      "config_var":"hostname"
+      "Database name": "traffic_ops_db",
+      "config_var": "dbname"
     },
     {
-      "LDAP Admin DN":"",
-      "config_var":"admin_dn"
+      "Database server hostname IP or FQDN": "localhost",
+      "config_var": "hostname"
     },
     {
-      "LDAP Admin Password":"",
-      "config_var":"password",
-      "hidden":"1"
+      "Database port number": "5432",
+      "config_var": "port"
     },
     {
-      "LDAP Search Base":"",
-      "config_var":"search_base"
+      "Traffic Ops database user": "dbuser",
+      "config_var": "user"
+    },
+    {
+      "Traffic Ops database password": "dbpass",
+      "config_var": "password",
+      "hidden": "1"
```

Comment:
Can we add something to the deprecation? The Perl version of `_postinstall` only supports `hidden` properties as strings containing an integer encoding a boolean concept (I think?), but in Python it'll happily work with just booleans. And they really ought to be boolean. I've got other complaints about the format of these input JSON files, but that's the big one.

Reply:
I'd rather not make breaking changes in this PR. This is supposed to be a drop-in-replacement right now, not a better-thought-out version with breaking changes.

Reply:
I wasn't suggesting a breaking change, just a deprecation.

Reply:
Gotcha, deprecated in 71a84ca259.